src/main/java/tech/fastj/math/Maths.java

```diff
@@ -201,4 +201,15 @@ public static float denormalize(float normalizedNum, float min, float max) {
 
         return (normalizedNum * (max - min)) + min;
     }
+
+    /**
+     * Returns a {@code float} based on the value between a and b.
+     * 
+     * @param a Starting value.
+     * @param b Ending value.
+     * @param y The value between the starting and ending values. 
+     **/
+    public static float inverseLerp(float a, float b, float y) {
```

Comment:
These variable names are a bit vague -- give them names that more directly represent what they represent in the code.